package.json

```diff
@@ -53,7 +53,6 @@
     "ember-data": "^2.17.0",
     "ember-disable-prototype-extensions": "^1.1.2",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.0.0",
```

Comment:
removing this so that we're testing the local project, not the published package